src/Public/OpenApi.ps1

```diff
@@ -1219,6 +1219,55 @@ function New-PodeOAObjectProperty
     return $param
 }
 
+<#
+.SYNOPSIS
+Creates a OpenAPI schema reference property.
+
+.DESCRIPTION
+Creates a new OpenAPI schema reference from another OpenAPI schema.
+
+.PARAMETER Name
+The Name of the property.
+
+.PARAMETER ComponentSchema
+An component schema name.
+
+.PARAMETER Description
+A Description of the property.
+
+.EXAMPLE
+New-PodeOASchemaProperty -Name 'Config' -ComponentSchema "MyConfigSchema"
+#>
+function New-PodeOASchemaProperty
+{
+    [CmdletBinding()]
+    param(
+        [Parameter()]
+        [string]
+        $Name,
+
+        [Parameter(Mandatory=$true, ParameterSetName = 'ComponentSchema')]
+        [string]
+        $ComponentSchema,
+
+        [Parameter()]
+        [string]
+        $Description
+    )
+
+    if(-not (Test-PodeOAComponentSchema -Name $ComponentSchema)) {
+        throw "Could not find component schema $ComponentSchema";
```

Comment:
Nah, that one's on me, I think. I worked on multiple PS-projects yesterday and _my_ styleguide states "use `;`".
So that slipped.
(Perhaps an .editorconfig would be a nice?)